mozilla-release/browser/components/AutoForgetTabsService.js

```diff
@@ -94,7 +94,8 @@ AutoForgetTabsService.prototype = {
     }
   },
 
-  blacklisted: function AFTSvc_blacklisted(domain) {
+  blacklisted: function AFTSvc_blacklisted(domain, isURL) {
+    if (isURL) domain = maybeGetDomain(domain);
```

Comment:
@tsl143 You know, I think it is better to use curly braces here although source code in this file is written by us. Just to make it easier to pass FF lint validation in the future.